fighthealthinsurance/ml_models.py

```diff
@@ -216,6 +216,13 @@ def _clean_diagnosis_response(self, response):
             return None
         return self.diagnosis_response_regex.sub("", response)
 
+    @cache
+    def get_fax_number(self, denial: str) -> Optional[str]:
+        return self._infer(
+            system_prompt="You are a helpful assistant.",
+            prompt=f"Tell me what the to appeal fax number in the provided denial. If unknown write UNKNOWN. If known just without any pre-amble and as a snippet from the original doc. The denial follows: {denial}",
```

Comment:
We'll probably want to clean up the grammar here.

Should be: "Tell me the to appeal fax number _is_ within the provided denial. If the fax number is unknown write UNKNOWN. If known just output the fax number without any pre-amble and as a snipper from the original doc..."